added new roles and fixed typo (#15)
TTT-2/docs: docs/general/roles.md

    |**Survivalist**|You have the benefit of having a shop!|
    |**Trapper**|You may use Traitor buttons!|
    |**Wrath**|If another Innocent kills you, you will be brought back as a Traitor!|

>     |**Beacon**|You will get visbile stronger the more people are confirmed death!|

Reviewer: ```suggestion
    |**Beacon**|You will get stronger the more people are confirmed dead!|
```

>     |**Lycanthrope**|You will get very powerful if you are the last Innocent!|

Reviewer: ```suggestion
    |**Lycanthrope**|You will get very powerful, if you are the last Innocent!|
```

    |**Medium**|You can read a scrambled version of the Spectator chat!|

>     |**Liar**|Like the Wrath you don't know your role! Your corpse will be turned to a Traitor corpse with one credit!|

Reviewer: ```suggestion
    |**Liar**|Like the Wrath, you don't know your role! Your corpse can be turned to a Traitor corpse with one credit!|
```

    |**Totemhunter**|Destroy totems to earn credits!|
    |**Traitor**|The 'bad' guys, just like you know them!|
    |**Vampire**|May transform into a bat and heal himself after killing a player!|
    |**Executioner**|Has to kill specific targets, he also does more damage to their target and less to non-targets!|

>     |**Defective**|You are a fake Detective who is in reality a Traitor!|

Reviewer: ```suggestion
    |**Defective**|You are a fake Detective, who is a Traitor in reality!|
```

    |**Infected**|Has to kill everyone! Everyone he kills will become his zombie!|
    |**Jackal**|Has to kill everyone! Can make himself a Sidekick!|
    |**Necromancer**|Has to kill everyone! Can revive players as his zombies!|
    |**Ravenous**|Has to kill everyone! The Glutton becomes this if he gets too hungry!|
    |**Serialkiller**|Has to kill everyone! Also has a wallhack!|
    |**Restless**|Has to kill everyone! He can die multiple times but gets weaker!|

>     |**Doppelganger**|He needs to steal roles to take the appearance/function of these but he is still in his own team!|

Reviewer: ```suggestion
    |**Doppelganger**|He needs to steal roles to take the appearance/function of those but he is still in his own team!|
```

    |**Pirate**|You can give a player your contract and join their team!|
    |**Sidekick**|Help the one that made you a Sidekick win!|
    |**Unknown**|Has to die by another player and will become that players role!|
    |**Medic**|He needs to help everyone or choose a team to help! He has healing powers and he can't win alone!|
    |**Cursed**|He can't deal damage but he will always resurrect!|

>     |**Mimic**|He needs to steal roles to take the team/appearance and function of these!|

Reviewer: ```suggestion
    |**Mimic**|He needs to steal roles to take the team/appearance and function of those!|
```
